sheets/snippets/test/test_sheets_batch_update.js

```diff
@@ -0,0 +1,64 @@
+/**
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+const Promise = require('promise');
+const expect = require('expect');
+const Helpers = require('./helpers');
+const SheetsBatchUpdate = require('../sheets_batch_update');
+
+const mochaAsync = (fn) => {
```

Comment:
This isn't necessary -- mocha allows returning promises instead of calling done, so no need to wrap tests to convert from promises.

https://calendar.google.com/calendar/u/0/syncselect